kolibri/plugins/facility/assets/src/views/FacilityConfigPage/index.vue

```diff
@@ -218,23 +216,30 @@
           this.toggleSetting('learner_can_edit_password');
         }
       },
-      dismissNotification() {
-        this.$store.commit('facilityConfig/CONFIG_PAGE_NOTIFY', null);
+      updateSettings(action) {
+        this.$store
+          .dispatch(action)
+          .then(() => {
+            // eslint-disable-next-line kolibri/vue-no-undefined-string-uses
+            this.createSnackbar(this.NotificationsStrings.$tr('saveSuccess'));
```

Comment:
I think this change might be causing the test failure [here](https://travis-ci.org/github/learningequality/kolibri/jobs/731420697#L7788), because we're calling the `$store.dispatch` mock in the test a second time here in `createSnackbar` (under the hood, since the `createSnackbar` method created by `mapActions(['createSnackbar'])`is a shorthand for `store.dispatch('createSnackbar', args)`

Reply:
I think it would be fine to update the assertion that the mock was called twice, and add a new assertion to say the mock was called with the snackbar action.